README.md

```diff
@@ -27,8 +27,91 @@
 *Vue language client for coc.nvim*
 
 [neovim/nvim-lspconfig](https://github.com/neovim/nvim-lspconfig) ⚡ 🤝 \
-*Vue language server configuration for Neovim* \
-[[Volar 2.0 version set up tutorial](https://github.com/vuejs/language-tools/issues/3925)]
+*Vue language server configuration for Neovim*
+
+<details>
+  <summary>How to configure vue language server with neovim and lsp?</summary>
+
+## Configuration for versions:  `^2.0.0`
+
+Note: take over mode has been removed. You have to run `@vue/language-server` alongside a TypeScript server that is running `@vue/typescript-plugin`. Here is a minimal configuration for Neovim's LSP to make the language server work after upgrading to version `2.0.0`.
+
+  ```lua
+-- If you are using mason.nvim, you can get the ts_plugin_path like this
+-- local mason_registry = require('mason-registry')
+-- local ts_plugin_path = mason_registry.get_package('vue-language-server'):get_install_path() .. '/node_modules/@vue/language-server/node_modules/@vue/typescript-plugin'
+
+local ts_plugin_path = '/path/to/@vue/typescript-plugin'
+
+local lspconfig = require('lspconfig')
+
+lspconfig.tsserver.setup {
+  init_options = {
+    plugins = {
+      {
+        name = '@vue/typescript-plugin',
+        location = ts_plugin_path,
```

Comment:
As I said at https://github.com/vuejs/language-tools/issues/3925#issuecomment-1987733530, specifying location as @vue/language-server path should be able to directly reference @vue/language-server on which it depends. @vue/typescript-plugin, users do not need to install @vue/typescript-plugin globally and can ensure that it is the same version.

Does the documentation not suggest this because it doesn't work for you?

Reply:
Ops missed that, yes just tried and it worded. Thanks!

Reply:
@johnsoncodehk With both `@vue/language-server` and `@vue/typescript-plugin` installed globally (pnpm) there is nothing in `/Users/gegoune/.pnpm-global/5/node_modules/@vue/language-server/node_modules`. Doesn't that mean that I need to specify path to `/Users/gegoune/.pnpm-global/5/node_modules/@vue/typescript-plugin` in `location`?

Reply:
You should be fine with `npm install` I think pnpm uses it link magic so there's no `.../@vue/language-server/node_modules`